.github/workflows/labeler.yml

```diff
@@ -22,3 +24,21 @@ jobs:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           configuration-path: .github/labeler.yml
           enable-versioned-regex: 0
+
+      - name: Set workflowId environment variable
+        id: set_workflow_id
+        run: |
+          if [[ "${{ github.event.action }}" == "labeled" ]]; then
+            echo "workflowId=new-migrated-issue.yml" >> $GITHUB_ENV
+          else
+            echo "workflowId=delete-migrated-issue.yml" >> $GITHUB_ENV
+          fi
```

Comment:
Might be better to use outputs rather than environment variables here:

https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs